objc/runtime.h

```diff
@@ -26,6 +26,12 @@ extern "C" {
 #	endif
 #endif
 
+#if __has_attribute(ns_returns_retained)
```

Comment:
Should this be enclosed in `#ifndef OBJC_RETURNS_RETAINED`?

Reply:
I don't think so, this header shouldn't be included twice and it's not defined anywhere else.  The above macro is, because it's usually defined by clang.